src/main/java/com/mybookfinder/enterprise/dao/IBookDAO.java

```diff
@@ -2,5 +2,5 @@
 
 import com.mybookfinder.enterprise.dto.Book;
 public interface IBookDAO {
-    Book save(Book book);
+    Book save(Book book) throws Exception;
```

Comment:
Added "throws Exception" to save functions in order to catch potential exceptions. For example: What if the database being saved to is offline?
Source: Module 5 video TDDBDD Unit Test Example With Mockito and IntelliJ Idea